### ISSUE TITLE
Can't apply Hellfire enchantment to wand Focus Blink

### ISSUE THREAD

**Issue by TA0MACHINE** [GTNewHorizons/GT-New-Horizons-Modpack#7721]
Version 2.1.0.0 23.12.2020
multiplayer: Eta
The same issue like: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7655
Tried add Hellfire enchantment to wand Focus Blink in Focal Manipulator.
When you press "start upgrade" nothing happens.
I had 40 xp in that time (enchantment costs 16 xp).
TAO#4382
![9GvUx1](https://user-images.githubusercontent.com/42874669/112806505-8154d980-907f-11eb-81d5-7fb089e46713.png)

**Comment by repo-alt** [comment]
Most probable cause is upgrade id conflict. There should be a message in the logs about "Focus Upgrade id" in that case

**Comment by repo-alt** [comment]
Ok, I tested that in single player on zeta build and it works fine, so either it is fixed or it is Thermos

**Comment by the-bukkit** [comment]
I also have this problem.
I have tried to apply the following three upgrades to foci:

Illumination - Solar
![grafik](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/23724545/709a5dc2-eaa1-45d1-af04-3ee71545946b)

Blink - Pandemonium
![grafik](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/23724545/6e8bda51-06df-4f68-b69b-2c9ba3ecb231)

Blink - Hellfire
![grafik](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/23724545/3c9b15a0-605e-4960-826c-a80d791f8bf7)

But they also didn't start, despite there being a source of centi-vis and me having enough XP.

I have tested it on a new 2.5.1 server, and confirmed it only happens on servers, not singleplayer.
I also tried changing the "focus upgrades" id of the Pandemonium upgrade in the Forbidden Magic config, from 143 to 14300, but that didn't resolve the problem.
